### DISCUSSION TITLE
Markdown Oxide is now availiable on Zed!

### DISCUSSION THREAD

**Discussion by Feel-ix-343** [Feel-ix-343/markdown-oxide#37] [Announcements]
Install the Markdown Oxide extension to use it. Note that you must have the LS installed to `markdown-oxide` to use it.

**Comment by Feel-ix-343** [comment]
Especially try out references, definitions, hover, and completions. The multi-buffer references view is quite interesting!

**Reply by VKondakoff** [reply]
Any chance to see LS binaries releases?

**Reply by Feel-ix-343** [reply]
Yes we can set up some nightly binary releases. Do you have any experience with this? I don't but I can try

**Reply by VKondakoff** [reply]
No, sorry, no experience with this.
Here are two links:
https://github.com/marketplace/actions/build-and-upload-rust-binary-to-github-releases
https://dzfrias.dev/blog/deploy-rust-cross-platform-github-actions/
Any useful info here?

**Reply by Feel-ix-343** [reply]
Hey! Those links were helpful

I opened a PR to update the Zed extension to use binaries [here](https://github.com/zed-industries/extensions/pull/420)
